http/src/main/resources/jupyter/welder-docker-compose.yaml

```diff
@@ -13,6 +13,7 @@ services:
     environment:
       # TODO what env vars are actually needed here?
       GOOGLE_PROJECT: "${GOOGLE_PROJECT}"
+      STAGING_BUCKET: "${STAGING_BUCKET}"
```

Comment:
Should we set this in `jupyter-docker-compose` as well?

Reply:
That way we could verify the EV in an automation test (we have existing tests to verify EVs are set in notebooks).

Reply:
that's a bit awkward way of testing....cuz we're not actually validating the environment variables where it should be set (inside welder container)

Reply:
Agreed, just thinking it may be useful for notebooks users to have access to this EV as well.

Reply:
do users need to know about staging bucket?